jupyter/scripts/entrypoint.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+
+command=$1
+arguments=${*:2}
+if [[ -z $arguments ]]; then
+  arguments_debug="no arguments"
+else
+  arguments_debug="arguments ($arguments)"
+fi
+
+# echo "Running \"$command\" with $arguments_debug"
+
+show_help() {
+  echo """
+  Available commands:
+
+  notebook            : start notebook server
+  pipeline            : run pipeline in local or cloud mode (ex: pipeline cloudrun '{}')
+  
+  Any arguments passed will be forwarded to the executed command
+  """
+}
+
+case "$command" in
```

Comment:
add start-singleuser as well

Reply:
as "start-singleuser.sh" to not break https://github.com/BLSQ/openhexa-infra/blob/66b0745f852beab324e593c4848a9ec6bb2e43ea/terraform/releases/modules/notebooks/fuse_config.yaml#L26C8-L26C27

Reply:
There is no need to add it since in openhexa-infra we specify in k8s the command (== entrypoint in docker) to run